lib/tapioca/generator.rb

```diff
@@ -559,6 +560,38 @@ def compile_gem_rbi(gem)
       end
     end
 
+    sig { params(gem: Gemfile::Gem, generated_content: String).returns(String) }
+    def merge_with_copied_rbi(gem, generated_content)
+      return generated_content unless File.directory?("#{gem.full_gem_path}/rbi")
+
+      generated_rbi = RBI::Parser.parse_string(generated_content)
+      copied_rbi = copy_gem_rbi(gem)
+      display_conflict_warnings(generated_rbi, copied_rbi)
+      final_rbi = RBI::Rewriters::Merge.merge_trees(generated_rbi, copied_rbi, keep: RBI::Rewriters::Merge::Keep::LEFT)
+      final_rbi.string
+    end
+
+    sig { params(gem: Gemfile::Gem).returns(RBI::Tree) }
+    def copy_gem_rbi(gem)
+      rbi = RBI::Tree.new
+      Dir.glob("#{gem.full_gem_path}/rbi/*.rbi").each do |file|
+        gem_rbi = RBI::Parser.parse_file(file)
+        display_conflict_warnings(rbi, gem_rbi)
+        rbi = RBI::Rewriters::Merge.merge_trees(rbi, gem_rbi, keep: RBI::Rewriters::Merge::Keep::NONE)
+      end
+      rbi
+    end
+
+    sig { params(generated_rbi: RBI::Tree, copied_rbi: RBI::Tree).void }
+    def display_conflict_warnings(generated_rbi, copied_rbi)
+      rewriter = RBI::Rewriters::Merge.new
+      rewriter.merge(generated_rbi)
+      conflicts = rewriter.merge(copied_rbi)
+      indent do
+        say("\n#{conflicts.join("\n")}") unless conflicts.empty?
```

Comment:
even with using this indent command, I wasn't able to make it match the rest of the commands. Any suggestions?

Reply:
Do you have an example of how you want to display this?

Note: I'm not sure a simple white line is enough for the warning to be visible. The output is already verbose and the line might be lost in the middle of the other lines. You could be using some colours as seen here: https://github.com/Shopify/tapioca/blob/92f2a10cf7f603ec1a00d77ad7624c2735542d28/lib/tapioca/generator.rb#L52

Maybe even a prefix such has `<yellow+bold>Warning:</yellow+bold> rest of the line`.